config/main.py

```diff
@@ -406,7 +408,12 @@ def add_vlan_member(ctx, vid, interface_name, untagged):
     vlan = db.get_entry('VLAN', vlan_name)
     
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
```

Comment:
The introduction of the variable `default_name` is unnecessary. Simply assign directly to `interface_name` and compare it to None. You can then also eliminate the `else` clause:

```
interface_name = interface_alias_to_name(interface_name)
if interface_name is None:
    print "Invalid interface {}".format(interface_name)
    raise click.Abort()
```